index.html

```diff
@@ -187,7 +222,10 @@ <h2 class="h2">Get Started</h2>
                 <textarea type="text" placeholder="Enter your message" name="message" maxlength="500" class="message"  required></textarea>
                 <input type="submit" value="Start Collaboration" class="buttons contact__button">
```

Comment:
- [ ] Kindly fix the issues with the form:
 - The input fields are not aligned properly.
 - The labels of the input fields are missing.
 - The `font-family` of the input fields is not the same as the one in Figma.
 - The `height` of the textarea is also not the same.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200125228-d04f1b75-f7b6-49b3-848a-5fc9f85d0e05.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200125232-2fcf6bb9-be93-46d2-ac33-1c7338526609.png)

Reply:
- [ Optional ] Kindly fix the issues with forms as suggested  by the previous reviewer.